src/screens/ParadeMapScreen/Map.js

```diff
@@ -129,6 +135,7 @@ class Map extends Component<Props, State> {
           <Touchable
             onPress={this.moveToCurrentLocation}
             style={styles.touchable}
+            background={TouchableNativeFeedback.SelectableBackgroundBorderless()}
```

Comment:
According to the docs this is only available on API level 21+: https://facebook.github.io/react-native/docs/touchablenativefeedback.html#selectablebackgroundborderless

The app is available for devides on 16+. So we need to validate that this doesn't crash on older devices. We should be able to test this in an emulator.